metricbeat/module/system/diskio/diskstat_windows_helper.go

```diff
@@ -142,6 +145,7 @@ func enablePerformanceCounters() error {
 	if err = key.SetDWordValue("EnableCounterForIoctl", 1); err != nil {
 		return errors.Errorf("cannot create EnableCounterForIoctl key in the registry in order to enable the performance counters: %s", err)
 	}
+	logger.Info("The key with the name EnableCounterForIoctl has been added in the windows registry in order to enable the performance counters")
```

Comment:
Has it been added if it already existed? 😄 

Also, can you put the whole key path in, like HKLM\System...